File: openapi-client/dart-jaguar/lib/model/mail_contact.dart
import 'package:jaguar_serializer/jaguar_serializer.dart';


part 'mail_contact.jser.dart';

class MailContact {
   /* The email address */
  @Alias('email', isNullable: false,  )
  final String email;
   /* Optional contact name */
  @Alias('name', isNullable: false,  )
  final String name;
  

  MailContact(
      

{
    
     this.email = null,   this.name = null 
    
    }
  );

  @override
  String toString() {
    return 'MailContact[email=$email, name=$name, ]';
  }
}

@GenSerializer(nullableFields: true)
class MailContactSerializer extends Serializer<MailContact> with _$MailContactSerializer {

}

File: openapi-client/scala-gatling/src/gatling/scala/org/openapitools/client/model/MailContact.scala

package org.openapitools.client.model


case class MailContact (
    /* The email address */
    _email: String,
    /* Optional contact name */
    _name: Option[String]
)
object MailContact {
    def toStringBody(var_email: Object, var_name: Object) =
        s"""
        | {
        | "email":$var_email,"name":$var_name
        | }
        """.stripMargin
}
